### ISSUE TITLE
Use TS cross validation instead of single validation set

### ISSUE THREAD

**Issue by sjtrny** [sjtrny/forecast_dash#18]
https://scikit-learn.org/stable/modules/cross_validation.html#timeseries-cv

**Comment by sjtrny** [comment]
Closed by https://github.com/sjtrny/forecast_dash/commit/73dce2248eafe63d4d6b4c2c2b4e551b013afc41
